composer.json

```diff
@@ -1,6 +1,8 @@
 {
 	"name"       : "wp-coding-standards/wpcs",
 	"description": "PHP_CodeSniffer rules (sniffs) to enforce WordPress coding conventions",
+	"type"       : "phpcs-standard",
+	"version": "9.0",
```

Comment:
TBH, I have no idea. Was certainly not my intention to introduce that one here (and the formatting is off as well).
I'll adapt the pull request...